test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -55,3 +55,56 @@ Feature: Bind a single application to multiple services
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-2" should be changed to "True"
         And "nodejs-app" deployment must contain SBR name "binding-request-1"
         And "nodejs-app" deployment must contain SBR name "binding-request-2"
+
+        Scenario: Bind two backend services by creating 1 SBR to a single application
+        Given Imported Nodejs application "nodejs-app-2-services-1-sbr" is running
+        * The Custom Resource is present
+        """
+        apiVersion: postgresql.baiju.dev/v1alpha1
+        kind: Database
+        metadata:
+            name: internal-db
+            annotations:
+                servicebindingoperator.redhat.io/spec.host_internal_db: 'binding:env:attribute'
+        spec:
+            host_internal_db: internal.db.postgresql.baiju.dev
+        """
+        * The Custom Resource is present
+        """
+        apiVersion: postgresql.baiju.dev/v1alpha1
+        kind: Database
+        metadata:
+            name: external-db
+            annotations:
+                servicebindingoperator.redhat.io/spec.host_external_db: 'binding:env:attribute'
+        spec:
+            host_external_db: external.db.postgresql.baiju.dev
+        """
+        When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-2-services-1-sbr
+            spec:
+                application:
+                    name: nodejs-app-2-services-1-sbr
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    name: internal-db
+                -   group: postgresql.baiju.dev
+                    version: v1alpha1
+                    kind: Database
+                    name: external-db
+            """
+
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-2-services-1-sbr" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-2-services-1-sbr" should be changed to "True"
+        And "nodejs-app-2-services-1-sbr" deployment must contain SBR name "binding-request-2-services-1-sbr"
```

Comment:
we do not check if app is really running correctly.

Reply:
We're checking if the deployment has the env var or not.

Reply:
we are checking only the binding secret is referred in the deployment, but not if env var is really available within app process.